ctc_lib/container_tool_command_base.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ctc_lib
+
+import (
+	"github.com/GoogleCloudPlatform/runtimes-common/ctc_lib/flags"
+	"github.com/GoogleCloudPlatform/runtimes-common/ctc_lib/util"
+	"github.com/spf13/cobra"
+)
+
+type ContainerToolCommandBase struct {
+	*cobra.Command
+	Phase string
+}
+
+func (ctb *ContainerToolCommandBase) init() {
+	ctb.AddSubCommands()
+	ctb.AddFlags()
+}
+
+func (ctb *ContainerToolCommandBase) AddSubCommands() {
+	// Add version subcommand
+	ctb.AddCommand(VersionCommand)
```

Comment:
Just to double check - does this add a version subcommand for every command, or just the root?

Reply:
just the root